From 9fca96a2347650466126228bd9e5e7dbe19046e7 Mon Sep 17 00:00:00 2001
From: Chris Fries <cfries@motorola.com>
Date: Wed, 14 Oct 2015 23:00:29 -0500
Subject: [PATCH 393/959] IKSWM-7441: motoconfig: Enable INPUT_KEYCHORD

Enable INPUT_KEYCHORD driver to allow userspace init to detect
key chord events.

Change-Id: I76dea3c251cd0e0ad33d3695cc0c9e70b39da330
Signed-off-by: Chris Fries <cfries@motorola.com>
Reviewed-on: http://gerrit.mot.com/791669
SLTApproved: Slta Waiver <sltawvr@motorola.com>
SME-Granted: SME Approvals Granted
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Ravikumar Vembu <raviv@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index 1649ecfc181..d473ce99ebb 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -51,3 +51,4 @@ CONFIG_MSM_WATCHDOG_CTX_PRINT=y
 CONFIG_MSM_TZ_LOG=y
 CONFIG_MSM_TZ_LOG_WDOG_DUMP=y
 CONFIG_LEDS_ATC=y
+CONFIG_INPUT_KEYCHORD=y
-- 
2.11.0

